scripts/shared/clusters.sh

```diff
@@ -100,6 +100,7 @@ function create_kind_cluster() {
     ( deploy_cluster_capabilities; ) &
     if ! wait $! ; then
         echo "Failed to deploy cluster capabilities, removing the cluster"
+        kubectl cluster-info dump
```

Comment:
perhaps its better to dump to stderr?

Reply:
What's the difference on dumping to stderr?